cryptographic_estimators/SDEstimator/SDAlgorithms/bjmm_pdw.py

```diff
@@ -33,39 +42,32 @@
 class BJMMpdw(SDAlgorithm):
     def __init__(self, problem: SDProblem, **kwargs):
         """
-         Construct an instance of BJMM's estimator in depth 2 using partially disjoint
-         weight, applying explicit MitM-NN search on second level [MMT11]_,
-         [BJMM12]_, [EB22]_.
+        Construct an instance of BJMM's estimator in depth 2 using partially disjoint
+        weight, applying explicit MitM-NN search on second level [MMT11]_, [BJMM12]_, [EB22]_.
 
-         Expected weight distribution::
+        Expected weight distribution:
 
-             +--------------------------+--------------------+--------------------+--------+--------+
-             | <-+ n - k - l1 - 2 l2 +->|<-+ (k + l1) / 2 +->|<-+ (k + l1) / 2 +->|   l2   |   l2   |
-             |       w - 2 p - 2 w2     |         p          |         p          |   w2   |   w2   |
-             +--------------------------+--------------------+--------------------+--------+--------+
+        +--------------------------+--------------------+--------------------+--------+--------+
+        | <-+ n - k - l1 - 2 l2 +->|<-+ (k + l1) / 2 +->|<-+ (k + l1) / 2 +->|   l2   |   l2   |
+        |       w - 2 p - 2 w2     |         p          |         p          |   w2   |   w2   |
+        +--------------------------+--------------------+--------------------+--------+--------+
```

Comment:
in case we require the indentations, this has to be changed back (just marking in case)